sycl/test-e2e/InvokeSimd/lit.local.cfg

```diff
@@ -1,10 +1,7 @@
 import platform
 
-if config.sycl_be in ['ext_oneapi_cuda', 'ext_oneapi_hip']:
-   config.unsupported = True
-
-if 'gpu' not in config.target_devices.split(','):
-   config.unsupported = True
+config.unsupported_features += ['cuda', 'hip']
```

Comment:
The description looks a bit confusing to me. 
It says: 
"I've dropped support for the old deprecated spelling of some of the backends (cuda/hip/level_zero), use ext_oneapi_* spelling instead".
But here you actually use 'cuda' and 'hip'